README.md

```diff
@@ -54,23 +54,17 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://clever-crepe-68033d.netlify.app/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
-### Install
-```
-npm install
-```
-### Usage
-```
-npm start
-```
-```
-
```

Comment:
- [OPTIONAL] I don't why you deleted this part but it would be better if you explain how to run the project locally in your README file. I suggest you add this part but after the part about how to clone the project then this should follow 🙃 🙃 
- [OPTIONAL] Also I think it could be better if you add a good description of the project, this project is not about displaying a logo by using CSS but is for mathematics, operations,... 🙃 🙃